test/components/Auth/EnsureLoggedInTest.js

```diff
@@ -47,4 +49,78 @@ describe('EnsureLoggedIn', function test () {
     it('Should render children when no error is present and user is logged in', () => {
         expect(this.loggedInOutput.type).to.equal('div')
     })
+
```

Comment:
There's a missing context here, which I can tell because there is shared setup between two context or describe blocks. I believe its title is 'when the console has an error'

Reply:
The consensus seems to be that we are not going to test for propTypes.  Tests removed accordingly.  :)